lib/path.js

```diff
@@ -40,6 +40,15 @@ function assertPath(path) {
   }
 }
 
+function isPathSeparator(code) {
+  return code === CHAR_FORWARD_SLASH || code === CHAR_BACKWARD_SLASH;
+}
+
+function isWindowsDeviceRoot(code) {
+  return code >= CHAR_UPPERCASE_A && code <= CHAR_UPPERCASE_Z ||
+    code >= CHAR_LOWERCASE_A && code <= CHAR_LOWERCASE_Z;
```

Comment:
Nit: this would be slightly nicer aligned with the `code` part above, rather than just indented two spaces.